### DISCUSSION TITLE
监控节点或者边的变化，使用getNodes和getEdges，但是文档提示下个版本会被删除，让使用useVisibleEdges代替，但是代码里没到这个方法

### DISCUSSION THREAD

**Discussion by cc8992** [bcakmakoglu/vue-flow#1402] [Q&A] [answered]
![image](https://github.com/bcakmakoglu/vue-flow/assets/17571309/70737325-4dc8-44f6-a5fb-bfa84ed1339d)

**Comment by bcakmakoglu** [comment, marked as answer]
Sorry ignore that message for now, it's not supposed to be in there *yet* but will be in the future 😅
